app/frontend/hooks/use-search.ts

```diff
@@ -51,5 +51,5 @@ export const useSearch = (searchModel: ISearch, dependencyArray: any[] = []) =>
       page: searchModel.currentPage,
       countPerPage: searchModel.countPerPage,
     })
-  }, [...dependencyArray, location.search])
+  }, [...dependencyArray])
```

Comment:
@bkmaibach any specific reasons why `location.search` was a dependency here? I can't see any reason why the location search would change without a page refresh (which would trigger the useEffect anyway) unless this was added to fix something in particular?